src/Controllers/LoginSessionController.php

```diff
@@ -1,22 +1,18 @@
 <?php
 
-namespace SilverStripe\SessionManager\Control;
+namespace SilverStripe\SessionManager\Controllers;
 
 use Exception;
-use SilverStripe\Admin\LeftAndMain;
+use SilverStripe\Control\Controller;
 use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Control\HTTPResponse;
 use SilverStripe\Security\SecurityToken;
-use SilverStripe\SessionManager\Model\LoginSession;
+use SilverStripe\SessionManager\Models\LoginSession;
 
-// TODO: move this to Controllers folder and extend Controller instead of LeftAndMain
-class LoginSessionController extends LeftAndMain
+class LoginSessionController extends Controller
```

Comment:
@emteknetnz @bergice Changing this class from `LeftAndMain` to `Controller` completely nukes the functionality. I understand this is a big PR, but you should at least try to run these things before merging them.

Reply:
I'll fix it through https://github.com/silverstripe/silverstripe-session-manager/issues/67

Reply:
Sorry, it was my screw up. I change something else on my set up ... just ignore me.